theseus/embodied/collision/signed_distance_field.py

```diff
@@ -71,37 +82,75 @@ def _compute_sdf_data_from_map(
 
                 sdf_data = map_dist - inv_map_dist
                 # metric
-                sdf_data = torch.tensor(sdf_data, dtype=cell_size.dtype) * cell_size[i]
+                sdf_data = (
+                    torch.tensor(sdf_data, dtype=occupancy_map.dtype) * cell_size[i]
+                )
 
             all_sdf_data.append(sdf_data)
 
         sdf_data_var = Variable(torch.stack(all_sdf_data))
 
         return sdf_data_var
 
-    def update_data(self, origin: Variable, sdf_data: Variable, cell_size: Variable):
-        if sdf_data.ndim != 3:
-            raise ValueError(
-                "Argument sdf_data to SignedDistanceField2D must be a batch of matrices."
-            )
-        if not (origin.ndim == 2 or (origin.ndim == 3 and origin.shape[2] == 1)):
+    @staticmethod
+    def convert_origin(origin: Union[torch.Tensor, Point2]) -> Point2:
```

Comment:
Are these convert functions meant to be private and start with a trailing underscore?

Reply:
I'm also using these in the two collision cost functions.